roles/sap_hana_install/tasks/pre_tasks/check_version.yml

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+
+# Check if existing installation has expected version.
+# manifest file is located at SAP_HANA_DATABASE/server/manifest
+- name: SAP HANA - Pre-Tasks - Check availability of manifest file
+  ansible.builtin.stat:
+    path: "{{ __sap_hana_install_fact_hdblcm_path }}/server/manifest"
+  register: __sap_hana_install_register_stat_manifest
+
+# Execute version checks only if manifest file is present.
+- name: Block for manifest file
+  when: __sap_hana_install_register_stat_manifest.stat.exists
+  block:
+    - name: SAP HANA - Pre-Tasks - Get contents of manifest file
+      ansible.builtin.slurp:
+        src: "{{ __sap_hana_install_fact_hdblcm_path }}/server/manifest"
+      register: __sap_hana_install_register_slurp_manifest
+
+    - name: SAP HANA - Pre-Tasks - Parse manifest file into facts
+      ansible.builtin.set_fact:
+        __sap_hana_install_fact_manifest_revision: "{{ __revision.split(':')[1] | trim | int }}"
```

Comment:
I tested with Ansible 2.16.14 and 2.18.9 and the type of `__sap_hana_install_fact_manifest_revision` was always `AnsibleUnsafeText`, so `__sap_hana_install_fact_is_lss_required` was never set in the task of line 45 even when using SPS08.
This is most likely related to https://github.com/netbox-community/ansible_modules/issues/985 and https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_core_2.19.html#native-jinja-mode-required .